js/data.js

```diff
@@ -0,0 +1,67 @@
+import {
+  getRandomPositiveInteger,
+  getRandomPositiveIntegerWithDecimalPlaces,
+  getRandomArrayElement,
+  getArrayWithNoRepeat,
+  getImageId} from './util.js';
+
+const TYPES = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const CHECKINS = ['12:00', '13:00', '14:00'];
+
+const CHECKOUTS = ['12:00',' 13:00', '14:00'];
+
+const FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+
+
+const similarAdList =[];
+
+const getSimilarAdList =(listLength)=>{
```

Comment:
```suggestion
const getSimilarAds = (listLength) => {
```

использовать в названии List, ну такое. Ставь существительное во множественном числе. 

Ну и внимательность)